wukong/request.py

```diff
@@ -27,105 +30,138 @@ class SolrRequest(object):
     """
     Handle requests to SOLR and response from SOLR
     """
-    def __init__(self, solr_hosts, zookeeper_hosts=None, timeout=15):
+
+    def __init__(self, solr_hosts, zookeeper_hosts=None, timeout=15, refresh_frequency=2):
```

Comment:
Can we add a doc string here telling people what types or units these should be?